sources/packages/backend/apps/api/src/route-controllers/student-appeal/student-appeal.institutions.controller.ts

```diff
@@ -42,10 +42,10 @@ export class StudentAppealInstitutionsController extends BaseController {
   async getStudentAppealWithRequests(
     @Param("studentId", ParseIntPipe) studentId: number,
     @Param("appealId", ParseIntPipe) appealId: number,
-  ): Promise<StudentAppealAPIOutDTO<DetailedStudentAppealRequestAPIOutDTO>> {
-    return this.studentAppealControllerService.getStudentAppealWithRequests<DetailedStudentAppealRequestAPIOutDTO>(
+  ): Promise<StudentAppealAPIOutDTO<StudentAppealRequestAPIOutDTO>> {
```

Comment:
Just double-checking, the previous implementation wrongly returned the details to the institution and this change is now making it right?

Reply:
yes, that correct